test/unit/specs/api/lifecycle_spec.js

```diff
@@ -112,7 +112,7 @@ describe('Lifecycle API', function () {
     })
 
     it('hooks', function () {
-      var hooks = ['created', 'beforeCompile', 'compiled', 'attached', 'ready']
+      var hooks = ['created', 'beforeCompile', 'compiled', 'beforeAttach', 'attached', 'ready']
```

Comment:
@vuchl it looked like this series of tests was for monitoring adding stuff to the DOM, rather the the removal aspect but I could be wrong...